manifests/addons/metrics-emitter.yml

```diff
@@ -0,0 +1,48 @@
+params:
+  cf.deployment_name: (( param "Please specify the deployment name in bosh for CF" ))
+
+instance_groups:
+- name:      rmq-broker
+  jobs:
+  - name: loggregator_agent
+    release: loggregator-agent
+    consumes:
+      doppler:
+        from: doppler
+        deployment: (( grab params.cf.deployment_name ))
+    properties:
+      loggregator:
+        tls:
+          ca_cert: (( vault meta.cf.exodus_path ":loggregator_ca" ))
+          agent:
+            cert: (( vault meta.cf.exodus_path ":loggregator_tls_agent_cert" ))
+            key: (( vault meta.cf.exodus_path ":loggregator_tls_agent_key" ))
+  - name: rabbitmq-metrics-emitter
+    release: rabbitmq-metrics-emitter
+    properties:
+      rabbitmq_metrics_emitter:
+        cloud_foundry:
+          api:    (( grab meta.cf.api_url ))
+          skip_ssl_validation: (( grab meta.cf.skip_ssl_validation ))
+          username:   (( vault meta.cf.exodus_path ":admin_username" ))
+          password:   (( vault meta.cf.exodus_path ":admin_password" ))
+        rmq_management:
+          endpoint: (( concat meta.mgmt_scheme "://" params.rmq_domain ":" meta.mgmt_port "/api" ))
```

Comment:
change the domain portion of this to reference `mgmt_domain` instead of `rmq_domain`. They'll usually be the same, but not necessarily.